### ISSUE TITLE
Pin designation for BL Touch

### ISSUE THREAD

**Issue by Muffinfreak2** [stesta/CrealityEnder5Pro-SonicPad#1]
Hey,

i used the firmware you provided and its working great exept i cant get the Z axis to Home. I tried switching the Pins in the code for the Probe and on the Z axis but nothing i working. I am pretty sure that my BL Touch the problem is because you say you use a CR Touch. I cant find a place where i can find the pin designation for a BL Touch.

Can you help me with that?

Thank you

**Comment by stesta** [comment]
@Muffinfreak2 I haven't used the BLTouch before. I did a little bit of digging to see if there was anyone else out there using a 4.2.7 board with the BLTouch specifically. I found one example here... https://gist.github.com/jasonehines/51b3bec00d6efdb23a006afa6969b105#file-printer-conf-L58-L72 

I noticed that the pins seem to be the same, but some of the probe touch modes are different. Maybe one of those will do the trick for you?

Let me know how it goes!
